recipes-debian/libnss-mdns/libnss-mdns_debian.bb

```diff
@@ -0,0 +1,47 @@
+# base recipe: meta/recipes-support/libnss-mdns/libnss-mdns_0.10.bb
+# base branch: warrior
+# base commit: 38d5c8ea98cfa49825c473eba8984c12edf062be
+
+SUMMARY = "Name Service Switch module for Multicast DNS (zeroconf) name resolution"
+HOMEPAGE = "http://0pointer.de/lennart/projects/nss-mdns/"
+SECTION = "libs"
+
+LICENSE = "LGPLv2.1+"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2d5025d4aa3495befef8f17206a5b0a1"
+
+DEPENDS = "avahi"
+PR = "r7"
+PV = "0.14.1"
+
+inherit autotools debian-package
+require recipes-debian/sources/nss-mdns.inc
+
+DEBIAN_PATCH_TYPE = 'quilt'
+DEBIAN_UNPACK_DIR = "${WORKDIR}/nss-mdns-${PV}"
+
+SRC_URI += "file://do-not-check-check.diff"
+
+localstatedir = "/"
+
+EXTRA_OECONF = "--libdir=${base_libdir} --disable-tests"
```

Comment:
How about add ``` --disable-lynx ``` as poky does?

Reply:
This was also removed in v0.11. See https://github.com/lathiat/nss-mdns/commit/0a49bf9239f58fdc2caa48f298cf3f0e4d96fd30.

Reply:
okay.